.github/workflows/mattermost.yml

```diff
@@ -0,0 +1,41 @@
+# This is a basic workflow to help you get started with Actions
```

Comment:
Please add the t8code licence statement.

Reply:
Thanks for the reminder!